db/internal_stats.cc

```diff
@@ -1472,6 +1513,8 @@ void InternalStats::DumpDBMapStats(
   }
   double seconds_up = (clock_->NowMicros() - started_at_) / kMicrosInSec;
   (*db_stats)["db.uptime"] = std::to_string(seconds_up);
+
+  DumpDBMapStatsWriteStall(db_stats);
```

Comment:
Is this redundant with the addition to db_stats_type_to_info?

Reply:
Oh right we remove the sum stats in DumpDBMapStatsWriteStall now they are the same. Removed it here.

Reply:
Fixed